.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": false,
+    "singleQuote": true
```

Comment:
It should be double quote? The complete project has double quote ig

Reply:
Yeah will keep double quotes ig from now on